resources/qml/SidebarHeader.qml

```diff
@@ -380,74 +435,6 @@ Column
         }
     }
 
-    // Material info row
```

Comment:
I suppose removing this is part of the design?

Reply:
Let's remove it and give it a try. If people complain we can always put it back.